common/psbt_open.c

```diff
@@ -494,20 +494,31 @@ bool psbt_output_to_external(const struct wally_psbt_output *output)
 	return !(!result);
 }
 
-bool psbt_contribs_changed(struct wally_psbt *orig,
+bool psbt_contribs_changed(const struct wally_psbt *orig,
```

Comment:
I think **both** arguments to `psbt_contribs_changed` should be const.  They are linearized internally to facilitate normalized comparison, but I don't think a caller would expect either argument to be modified.

Specifically, VLS is having trouble with the **2nd** argument being modified, not the first so this fix won't work.

Reply:
I put this as FIXME because we should change some logic down how.

An easy solution is to clone the input and output while we are sorting, without cloning all the PSBT